src/listr/tasks/platform/initTaskFactory.js

```diff
@@ -88,15 +91,34 @@ function initTaskFactory(
             dpnsDocumentSchema, ctx.identity,
           );
 
-          await ctx.client.platform.contracts.broadcast(
+          // TODO You need to make broadcast return ST
+          const stateTransition = await ctx.client.platform.contracts.broadcast(
             ctx.dataContract,
             ctx.identity,
           );
+          // Get state transition commit block height from Tenderdash
+          const tendermintRpcClient = createTenderdashRpcClient();
+
+          let stateTransitionHash = crypto.createHash('sha256')
+            .update(stateTransition.toBuffer())
```

Comment:
Do we need `toBuffer` here?

Reply:
Yep, https://github.com/dashevo/js-dash-sdk/pull/146